config/environments/production.template.rb

```diff
@@ -71,10 +71,24 @@
   # with SQLite, MySQL, and PostgreSQL)
   # config.active_record.auto_explain_threshold_in_seconds = 0.5
 
+  # Provide context to the email generator about the host
+  #config.action_mailer.default_url_options = {protocol: 'http', host: 'example.com' }
+
   config.action_mailer.delivery_method = :sendmail
   config.action_mailer.perform_deliveries = true
   config.action_mailer.raise_delivery_errors = true
 
+  # Use a custom smtp server, like Mandrill
+  #config.action_mailer.smtp_settings = {
+  #  address:              'smtp.mandrillapp.com',
+  #  port:                 25,
+  #  enable_starttls_auto: true,
+  #  authentication:       'login',
+  #  user_name:            'MANDRILL_USERNAME',
+  #  password:             'MANDRILL_PASSWORD',
+  #  domain:               'example.com',
+  #}
+
```

Comment:
I added some stock (optional) configuration to make it easier for people to add support for a custom mailer. You can see how to use it [here][1] and the [documentation I got it from][2], that @andrewwuan (I think) initially wrote.

[1]: https://github.com/autolab/Autolab/wiki/Deploying-Autolab-with-Docker#71-configure-email-with-mandrill
[2]: https://github.com/autolab/Autolab/wiki/Adding-email-support